cfy_manager/components/manager/manager.py

```diff
@@ -47,6 +47,14 @@ def _create_cloudify_user():
     common.mkdir(constants.CLOUDIFY_HOME_DIR)
 
 
+def _create_run_dirs():
```

Comment:
error handling if dir exists?

Reply:
Not needed — we use `common.mkdir` there, which uses the shell to create directories (using `mkdir -p`) rather than Python.